김민진/Bank/src/Bank/Main.java

```diff
@@ -0,0 +1,19 @@
+package Bank;
+
+public class Main {
+	
+	public Main() {
+		
+	}
+	
+	public void run() {
+		BankApplication bankApplication = new BankApplication();
```

Comment:
객체지향을 공부할 때 객체에게 어떤 행위를 하라고 메시지를 던지는 것인데, 해당 코드는 객체를 생성만 하고 있습니다.
어떤 행위를 할지 메시지를 던져주세요.
```suggestion
		BankApplication bankApplication = new BankApplication();
		bankApplication.run();
```